opensync-ext-cloud/src/main/java/com/telecominfraproject/wlan/opensync/external/integration/OpensyncExternalIntegrationMqttMessageProcessor.java

```diff
@@ -1225,7 +1188,7 @@ protected void processRealTimeSipCallStopEvent(int customerId, long equipmentId,
 
             if (apCallStop.hasClientMac()) {
 
-                cloudSipCallStopEvent.setClientMacAddress(MacAddress.valueOf(apCallStop.getClientMac()));
+                cloudSipCallStopEvent.setClientMacAddress(MacAddress.valueOf(apCallStop.getClientMac().toStringUtf8()));
```

Comment:
if you want to use the toStringUtf8, you should do the check for validity, first. i.e. isValidUtf8().
Otherwise consider using toByteArray(), which will work nicely in the MacAddress.valueOf as well? In your Unit tests you are building the opensync stats macAddress from an array of byte[] anyway